docs/getting-started/detections-req.asciidoc

```diff
@@ -100,10 +100,14 @@ a|The `manage`, `write`,`read`, and `view_index_metadata` index privileges for t
 
 ^1^ *NOTE*: If you're upgrading to {stack} 8.0.0 or later, users should have privileges for the `.alerts-security.alerts-<space-id>` AND `.siem-signals-<space-id>` indices. If you're newly installing the {stack}, then users do not need privileges for the `.siem-signals-<space-id>` index.
 
-| {kib} space `All` privileges for the `Security` feature (refer to
+a| {kib} space `All` privileges for the `Security` feature (refer to
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges])
 
-*NOTE*: To manage rule connectors, your role will also need `All` privileges for the `Action and Connectors` feature (*Management* -> *Action and Connectors*).
+*NOTE:* Your role will need additional `Action and Connectors` feature privileges (**Management → Action and Connectors**) to manage rules with actions and connectors:
+
+* To provide full access to editing rule actions and managing connectors, give your role `All` privileges. With `Read` privileges, your role can still edit rule actions but will have limited connector management. For example, your role can add or remove an existing connector from the rule, but it cannot create a new one.
```

Comment:
```suggestion
* To enable full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, your role can edit rule actions but has limited connector management capabilities. For example, `Read` privileges would allow you to add or remove an existing connector from the rule, but not to create a new one.
```

Reply:
Good with all of these, but am leaning towards keeping "provide" over "enable" since the latter makes me think of a toggle or a binary setting.